app/src/pages/inside/stepPage/modals/makeDecisionModal/makeDecisionModal.jsx

```diff
@@ -133,11 +156,46 @@ const MakeDecision = ({ data }) => {
       };
     });
   };
+  const sendSuggestResponse = () => {
+    if (modalState.decisionType === 'machineLearningSuggestions') {
+      const dataToSend = modalState.suggestedItems.map((item) => {
+        if (modalState.source.id === item.testItemResource.id) {
+          return {
+            ...item.suggestRs,
+            userChoice: 1,
+          };
+        }
+        return item.suggestRs;
+      });
+      fetch(URLS.choiceSuggestedItems(activeProject), {
+        method: 'put',
+        data: dataToSend,
+      })
+        .then(() => {
+          dispatch(
+            showNotification({
+              message: formatMessage(messages.suggestedChoiceSuccess),
+              type: NOTIFICATION_TYPES.SUCCESS,
+            }),
+          );
+        })
+        .catch(() => {
+          dispatch(
+            showNotification({
+              message: formatMessage(messages.suggestedChoiceFailed),
+              type: NOTIFICATION_TYPES.ERROR,
+            }),
+          );
+        });
+    }
+  };
   const saveDefect = (options) => {
     const { fetchFunc } = data;
     const issues = prepareDataToSend(options);
     const url = URLS.testItems(activeProject);
 
+    sendSuggestResponse();
```

Comment:
Looks like this function call every time? Maybe better solution is extract `decisionType===MACHINE_LEARNING_SUGGESTIONS` to here and leave one responsibility to function `sendSuggestResponse`